README.md

```diff
@@ -46,6 +46,8 @@ asdf_nodejs_version: '6.11.2'
 
 ```yaml
 - hosts: servers
+  environment:
+    PATH: /home/deploy/.asdf/bin:/home/deploy/.asdf/shims:{{ ansible_env.PATH }}
```

Comment:
The readme should indicate that the path be based on the `asdf_user` variable, maybe it could even be interpolated in?

Reply:
We can interpolate, this is definitely better.